src/CentreonModule/Application/DataRepresenter/UpdateAction.php

```diff
@@ -76,7 +76,7 @@ public function __construct(Module $entity = null, string $message = null)
      *
      * @return array<string,\CentreonModule\Application\DataRepresenter\ModuleEntity|string|null>
      */
-    public function jsonSerialize()
+    public function jsonSerialize(): mixed
```

Comment:
according to php this should be array (or phpdoc is incorrect)

Reply:
jsonSerialize signature has been updated in php 8.1
But return type of this method is still an array